packages/components/cascader/src/cascader.vue

```diff
@@ -40,7 +40,8 @@
           :disabled="isDisabled"
           :validate-event="false"
           :size="realSize"
-          :class="nsCascader.is('focus', popperVisible)"
+          :class="inputClass"
+          :tabindex="filterable ? '-1' : '0'"
```

Comment:
```suggestion
          :tabindex="multiple && filterable && !isDisabled ? -1 : undefined"
```

It should be ensured that only one input can be focused